cirq-core/cirq/experiments/two_qubit_xeb.py

```diff
@@ -386,3 +389,63 @@ def parallel_two_qubit_xeb(
     )
 
     return TwoQubitXEBResult(fit_exponential_decays(fids))
+
+
+def run_rb_and_xeb(
+    sampler: 'cirq.Sampler',
+    qubits: Optional[Sequence['cirq.GridQubit']] = None,
+    repetitions: int = 10**3,
+    num_circuits: int = 20,
+    num_clifford_range: Sequence[int] = tuple(
+        np.logspace(np.log10(5), np.log10(1000), 5, dtype=int)
+    ),
+    entangling_gate: 'cirq.Gate' = ops.CZ,
+    depths_xeb: Sequence[int] = tuple(np.arange(3, 100, 20)),
+    xeb_combinations: int = 10,
+    random_state: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = 42,
```

Comment:
Please replace the random_state default value with `None` here and in parallel_two_qubit_xeb.
As it is the randomized benchmarking is not random.

Reply:
As a user I would expect that RNG-dependent method would use a random seed just like stock builtin and numpy random functions, for example, `python -c "import random; print(random.random())"`.
The convention in `numpy` and the `random` module is to be not reproducible, which I feel makes sense for random processes.
All other cirq functions that take random_state argument use the `None` default if they have one.
I am not sure there is a strong enough reason to introduce a different behavior here.

Passing the `random_state` should achieve reproducibility, but that should be an opt-in behavior (by user providing that value) rather than the default.

We can look in a next PR if we can pass the random_state to the multiprocessing jobs to achieve exact reproducibility.

Reply:
done